### ISSUE TITLE
Make the command shorter and easier to remember

### ISSUE THREAD

**Issue by hotohoto** [pypyr/pypyr#312]
Or we may have an alias.

The command names of the other projects.
`poe`
`doit`
`npm`
`make`
`yarn`
`mvn`
`ant`
...

To me `pypyr` sounds a little too long and not easy to remember especially for persuading the other team members to use.

🙂

**Comment by yaythomas** [comment]
Thanks @hotohoto! Did you have a particular abbreviation in mind for an alias?

**Comment by hotohoto** [comment]
Not really. As a non-native speaker, I'm not quite sure what would be nice. Anyway,,
I'd say...`p` (and we may make it customizable in case it conflicts with another alise a user might already be using) or may be `ppr` like `mvn` or `npm` doesn't sound meaningful that much but it's short.
I guess it would be better to hear more opnions. Or just feel free to close this issue. I'm not super serious at the moment.😅

**Comment by yaythomas** [comment]
hehe, I'm pretty lazy with typing myself, so I admit I was a bit surprised to hear someone who wants even _shorter_ than `pypyr` 😆 

I don't know if `ppr` or `pyr` or `pp` (lol) is really all that much more memorable and functionally shorter than pypyr?

That said, I'm not against the idea and happy to hear further thoughts!

Making it customisable is not so easy from the pypyr installer's perspective - we'd have to pick the sorter aliases in advance, it wouldn't be user configurable.

While we think this through, in the meantime you and your team can use console aliases to make the command whatever you want it to be. . . If you're on zsh, this is how: https://zsh.sourceforge.io/Intro/intro_8.html. Bash follows much the same principle if you want to configure an alias there instead.

I'm going to move this off to discussions where we can continue throwing some ideas around!
